app/views/applications/deploy.html.erb

```diff
@@ -18,13 +18,6 @@
     <p class="lead add-top-margin">Production is <span class="label label-danger">not deployed yet</span>!</p>
   <% end %>
 
-  <% unless @application.staging_and_production_in_sync? %>
```

Comment:
> I'm not sure about removing this, it's a helpful to see something during a deploy. I think it might be confusing that this doesn't show up anymore?

Ok, so the main advantages I can think of are:
  - Showing something about the state of staging
    - The relevant page now only shows the state of production, whereas this gave some indication about staging
  - An extra indicator to help show if a deploy has already begun

> >   So it's expected that the environments are not in sync.
> 
> This is only true once you've deployed to staging, not when you initially start the deploy. I think it would still be useful to have an indication about what stage your deploy is in.

Yeah, I'll try and see if I can make some additional changes to improve these areas.